.eslintrc

```diff
@@ -1,25 +1,46 @@
-extends: idiomatic
-env:
-  node: true
-  es6: true
-  amd: true
-rules:
-  no-multi-spaces: 0
-  space-in-parens: 0
-  array-bracket-spacing: 0
-  computed-property-spacing: 0
-  key-spacing: 0
-  no-var: 2
-  indent:
-    - 2
-    - 2
-    -
-      VariableDeclarator:
-        var: 2
-        let: 2
-        const: 3
-  brace-style:
-    - 2
-    - 'stroustrup'
-    -
-      allowSingleLine: false
+{
+  "extends": [
+    "eslint:recommended",
+    "idiomatic"
+  ],
+  "parserOptions":{
+    "ecmaVersion": 2017,
+    "sourceType": "module"
+  },
+  "env":{
+    "node": true,
+    "es6": true
+  },
+  "rules": {
+    "no-multi-spaces": 0,
+    "space-in-parens": 0,
+    "array-bracket-spacing": 0,
+    "computed-property-spacing": 0,
+    "key-spacing": 0,
+    "no-var": 2,
+    "indent": [2, 2, {
+        "VariableDeclarator": {
+          "var": 2,
+          "let": 2,
+          "const": 3
+        }
+      }
+    ],
+    "brace-style": [2, "stroustrup",
+      {
+        "allowSingleLine": false
+      }
+    ],
+    "one-var":[2, "never"],
+    "semi": [2, "never"]
+  },
+  "overrides": [
+    {
+      "files":["./**/ampersand/*.js"],
+      "rules": {
+        "indent": [1, 4],
+        "semi": [1, "always"]
+      }
+    }
+  ]
+}
```

Comment:
Sorrynotsorry but my editor just didn't want to accept a YML-fied file. 

Also, not sure we have _all_ our rules here, but these are the main two that have bugged me for the entire time, so they're the ones we get until we make our own ESLint stuff.

Reply:
Have tried updating this again to use the yaml-ified file provided by the original repo, but no luck; instead this is now `.eslintrc.json` to prevent issues with conflicts (hopefully? i dunno)